src/components/Records/Search/Input/FilterAutocomplete.vue

```diff
@@ -133,65 +133,25 @@ export default {
     reset: function (selectedItem) {
       selectedItem.filterSelected = {};
     },
+    /**
+     * Call callOut method (an emit function to call scrollToBottom method from parent component) after a few seconds if last autocomplete item is focused
+     */
+    onInput() {
+      if (this.lastItem) {
+        setTimeout(this.callOut, 100)
+      }
+    },
+    /**
+     * Call an emit function name LastItemClick which trigger parent method name scrollToBottom
+     */
+    callOut() {
+      this.$emit('lastItemClick')
+    }
```

Comment:
Why I would do is:
- add an ID to the container of each autocomplete menu (eg. ` id='autocompleteLicenceList'`).
- use vue-scroll-to on each autocomplete field on focus when that scroll to the corresponding ID.
You wouldn't have to emit anything or create a callback.